frontend/src/newConsole/components/newLogos/LogoSidebar.tsx

```diff
@@ -1,116 +0,0 @@
-import * as React from 'react';
```

Comment:
@spokerman12  Can you simply use this file instead of creating a new ``LogosSidebar`` component? ``LogoSidebar`` has been already reviewed once part of another PR.

In that way, it would be easy to check your changes.

Reply:
The names for the components are wrong. See the filename `LogoSidebar`, then the exported component name `LogosSideBar` and others. So I based myself on the old `Logos` component and that `Sidebar` is a single word.

I'll restore the previous names.

Reply:
Oh, that makes, sense. Thanks